src/request.ts

```diff
@@ -891,13 +891,26 @@ class DatastoreRequest {
     query: Query,
     options: RunQueryStreamOptions = {}
   ): SharedQueryOptions {
+    // TODO: This needs to be refactored later:
```

Comment:
is this to be done in this PR or in a future one?

Reply:
Same the above comment. We can do it now if preferred.